pubsub/gossipsub/gossipsub-v1.0-old.md

```diff
@@ -0,0 +1,409 @@
+# gossipsub: An extensible baseline pubsub protocol
```

Comment:
Do we really want the doc to have `-old` in the name? AFAIU it is still valid.

Reply:
nvm. This is really the "old" :)

Reply:
perhaps `-r1`?